ui/components/Navbar/Navbar.styles.js

```diff
@@ -215,36 +212,48 @@ export const useStyles = makeStyles((theme) => ({
     },
   },
   collapseButtonWrapper: {
+    boxShadow:
+      "0.5px 0px 0px 0px rgb(0 0 0 / 20%), 1.5px 0px 0px 0px rgb(0 0 0 / 14%), 2.5px 1px 3px 0px rgb(0 0 0 / 12%)",
+    borderRadius: "0 5px 5px 0",
+    backgroundColor: "#FFFFFF",
+    color: "#263238",
```

Comment:
Declare what colour this is in this [file](https://github.com/meshery/meshery/blob/meshery-ui-restructuring/ui/styles/colors.js) so that we can reuse it.

Reply:
what should i name it?? @sudo-NithishKarthik

Reply:
I think it is `JungleGreen` or something.. 
@leecalcote , your help needed.